core/new-gui/package.json

```diff
@@ -17,7 +17,8 @@
     "e2e": "ng e2e",
     "preinstall": "cd multi-target && yarn install && yarn run build",
     "multi-target-updater": "cd multi-target && yarn run build && cd .. && yarn install ./multi-target -D",
-    "postinstall": "node git-version.js && ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points && yarn --cwd node_modules/@danmarshall/deckgl-typings run postinstall && node ./decorate-angular-cli.js"
+    "postinstall": "node git-version.js && ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points && yarn --cwd node_modules/@danmarshall/deckgl-typings run postinstall && node ./decorate-angular-cli.js",
+    "start-remote": "ng serve --host 0.0.0.0 --disable-host-check"
```

Comment:
what is the purpose of this command? please add some PR descriptions or comments in code so we can review it...

Reply:
The `run` command is not working on windows. This is a temporary fix. I will replace it.

Reply:
Could you add this to PR description and elaborate why `run` does not work?
And also please note this is a temporary fix in the PR title.

Reply:
Added. Found a good solution.